### ISSUE TITLE
Deployment getting stuck at processing data

### ISSUE THREAD

**Issue by nitin-neural** [Azure-Samples/azure-search-openai-demo#1521]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> Followed all the steps with both Container and Codespaces option. Deployment gets stuck at prepdocs step :

Ensuring search index gptkbindex exists
Creating gptkbindex search index
Ingesting 'role_library.pdf'
Extracting text from './data/role_library.pdf' using Azure Document Intelligence

![image](https://github.com/Azure-Samples/azure-search-openai-demo/assets/145154186/2113fd09-ddbf-4789-9c35-a87eedae857f)

I can see the metrics in Azure Document Intelligence as well

### Any log messages given by the failure
>

### Expected/desired behavior
>

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### azd version?
> run `azd version` and copy paste here.

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by pamelafox** [comment]
The ModuleNotFound error is often seen when the App Service deployment isn't quite ready. That's a frequent issue since azd currently reports a deployment is done before it's actually done. The azd and App Service teams are working on that. Typically that error resolves itself in 10 minutes if the deployment is successful, but sometimes a deployment fails.
I have more details in this doc: https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/docs/appservice.md

But if it's truly stuck at "Extracting text from './data/role_library.pdf' using Azure Document Intelligence", then it would never make it to the deploying stage. If it's stuck there, then I suspect a network issue when it's trying to upload that document to Document Intelligence. Has it ever succeeded with that call?

**Comment by nitin-neural** [comment]
> The ModuleNotFound error is often seen when the App Service deployment isn't quite ready. That's a frequent issue since azd currently reports a deployment is done before it's actually done. The azd and App Service teams are working on that. Typically that error resolves itself in 10 minutes if the deployment is successful, but sometimes a deployment fails. I have more details in this doc: https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/docs/appservice.md
> 
> But if it's truly stuck at "Extracting text from './data/role_library.pdf' using Azure Document Intelligence", then it would never make it to the deploying stage. If it's stuck there, then I suspect a network issue when it's trying to upload that document to Document Intelligence. Has it ever succeeded with that call?

yes if you see this document intelligence metrics screenshot attached below....more than 300 calls are successful....one file was uploaded but after 1 hour.  Strange thing is previously, I have deployed this repo multiple times ( last time I did was in early March) and I never got any of these issues. 

![321618773-2113fd09-ddbf-4789-9c35-a87eedae857f](https://github.com/Azure-Samples/azure-search-openai-demo/assets/145154186/d6cbe297-6a10-499e-a2b0-0009c3d242c2)
